homeassistant/components/xiaomi_miio/humidifier.py

```diff
@@ -232,7 +232,7 @@ def target_humidity(self):
         """Return the target humidity."""
         return (
             self._target_humidity
-            if self._mode == AirhumidifierOperationMode.Auto.name
+            if self._mode == AirhumidifierOperationMode.Auto.value
             or AirhumidifierOperationMode.Auto.name not in self.available_modes
```

Comment:
Should line 236 `name` not be changed to `value` as well?

Reply:
No because `self.available_modes` is a list of modes from backend library (names with a capital letter).

Reply:
Right we have
```python
   @property
    def mode(self):
        """Return the current mode."""
        return AirhumidifierOperationMode(self._mode).name
```